src/App.js

```diff
@@ -1,19 +1,51 @@
 import React from 'react';
 import AppPanel from './components/AppPanel';
+import ConfigPanel from './components/ConfigPanel';
 
-function App() {
-  return (
-    <div className="app">
-      <div className="app-panels">
-        <AppPanel type="config">
-          Config
-        </AppPanel>
-        <AppPanel type="preview">
-          Preview
-        </AppPanel>
+class App extends React.Component {
```

Comment:
Functional components w/ hooks is all the rage right now lol. I don't mind sticking to class based components right now though.

Reply:
Yeah let's refactor later